lib/business/data/becs.yml

```diff
@@ -0,0 +1,35 @@
+working_days:
+  - monday
+  - tuesday
+  - wednesday
+  - thursday
+  - friday
+
+holidays:
+  - January 1st, 2017
+  - January 2nd, 2017
+  - January 26th, 2017
+  - March 13th, 2017
+  - April 14th, 2017
+  - April 15th, 2017
+  - April 16th, 2017
+  - April 17th, 2017
+  - April 25th, 2017
+  - June 12th, 2017
+  - September 29th, 2017
+  - November 7th, 2017
+  - December 25th, 2017
+  - December 26th, 2017
+  - January 1st, 2018
+  - January 26th, 2018
+  - March 12th, 2018
+  - March 30th, 2018
+  - March 31th, 2018
+  - April 1st, 2018
+  - April 2nd, 2018
+  - April 25th, 2018
+  - June 11th, 2018
+  - September 28th, 2018
```

Comment:
It says TBD in `business.vic.gov.au`, is it safe to put this in here?

Reply:
I can't see `TBD` or anything indicating it isn't finalised. 
<img width="2" alt="screen shot 2017-12-05 at 17 51 40" src="https://user-images.githubusercontent.com/667610/34047578-9a4673e2-e1a8-11e7-8a6b-18b05775cbf8.png">

Reply:
Seems like `vic.gov.au` sometimes redirect's to a different version of the page with a `-2` or `-3` appended